src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,104 @@
+package duke;
```

Comment:
Perhaps you can put Parser in a different package inside the duke package, like duke.parser?